src/amr/solvers/solver_ppc.hpp

```diff
@@ -53,6 +59,10 @@ class SolverPPC : public ISolver<AMR_Types>
     Electromag electromagPred_{"EMPred"};
     Electromag electromagAvg_{"EMAvg"};
 
+    VecFieldT Bold_{this->name() + "_Bold", core::HybridQuantity::Vector::B};
+    VecFieldT fluxSumE_{this->name() + "_fluxSumE", core::HybridQuantity::Vector::E};
```

Comment:
this is a temporary?

there was a temporary added in PGL, so maybe we don't need this?